src/components/String/String.jsx

```diff
@@ -31,36 +31,32 @@ function Button({ stringNumber, note }) {
     setIsPlaying(false);
   }
 
+  function playAndStopSound() {
+    if (isPlaying) {
+      stopSound();
+    } else {
+      playSound();
+    }
+  }
+
   return (
-    <div className={styles.stringContainer}>
+    <button className={styles.stringContainer} onClick={playAndStopSound}>
       <div className={styles.stringSet}>
         <p className={styles.stringNumber}>
-          {getStringOrdinalFromNumber(stringNumber)} string {note}
+          {getStringOrdinalFromNumber(stringNumber)} string{" "}
+          <span className={styles.note}>{note}</span>
         </p>
-        <div className={styles.string}></div>
+        <div
+          role="button"
```

Comment:
Let's remove this role cos it's not a button. We already have our button element containing the whole string :)